qemu/tests/win_virtio_driver_install_by_installer.py

```diff
@@ -23,13 +19,15 @@ def run(test, params, env):
     7) Verify the qemu-ga version match expected version.
     8) Run driver signature check command in guest.
        Verify target driver.
+    9) Run driver function test.
 
     :param test: QEMU test object
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environment
     """
     run_install_cmd = params["run_install_cmd"]
     installer_pkg_check_cmd = params["installer_pkg_check_cmd"]
+    args = ast.literal_eval(params.get("args"))  # pylint: disable=W0611
```

Comment:
I think we can set a default value for `params.get()`,if so we can do not define it in cfg when we no extra parameters.

Reply:
^^